src/map.cpp

```diff
@@ -1305,6 +1305,32 @@ vehicle *map::veh_at_internal( const tripoint &p, int &part_num )
     return const_cast<vehicle *>( const_cast<const map *>( this )->veh_at_internal( p, part_num ) );
 }
 
+vehicle *map::find_vehicle( const tripoint_abs_ms &where ) const
```

Comment:
Why move this? It's kinda deceptive in `map` if it actually looks in the global mapbuffer. It was fine as a static function in vehicle.

(also, this function is pretty freaking crazy, but that's not on you)